src/test/scss/parser.test.ts

```diff
@@ -338,7 +539,7 @@ suite('SCSS - Parser', () => {
 
 	test('Nested Ruleset', function () {
 		let parser = new SCSSParser();
-		assertNode('.class1 { $let: 1; .class { $let: 2; three: $let; let: 3; } one: $let; }', parser, parser._parseRuleset.bind(parser));
+		assertNode('.class1 { $let: 1; .class { $let: 2; three: $let; let: 3; four: m.$foo } one: $let; }', parser, parser._parseRuleset.bind(parser));
```

Comment:
```suggestion
		assertNode('.class1 { $let: 1; .class { $let: 2; three: $let; let: 3; } one: $let; }', parser, parser._parseRuleset.bind(parser));
		assertNode('.class1 { $let: 1; .class { $let: m.$foo; } one: $let; }', parser, parser._parseRuleset.bind(parser));
```

Think it's better to keep the tests orthogonal.